src/utils.js

```diff
@@ -89,26 +89,26 @@ export function useNow(updateEvery = 1000) {
 export function useUniswapTokenRate(symbol) {
   const [tokenRates, setTokenRates] = useState(DEFAULT_RATE_STATE)
   const [tokenAddress] = getKnownContract(`TOKEN_${symbol}`)
-  const [antAddress] = getKnownContract(`TOKEN_ANT`)
+  const [anjAddress] = getKnownContract(`TOKEN_ANJ`)
```

Comment:
Renamed this to use anj data. I was thinking of just generalizing the hook to convert from symbol x -> symbol y.